### DISCUSSION TITLE
Can we implement filament charts like this? It seems like there's no specific content in the documentation.

### DISCUSSION THREAD

**Discussion by YANGJIAXUE2022510** [filamentphp/filament#8481] [Help]
![image](https://github.com/filamentphp/filament/assets/105269516/8d5bb400-a2d0-41a0-bdd1-a6a8a5e1daee)
Can we implement filament charts like this? It seems like there's no specific content in the documentation.

**Comment by leandrocfe** [comment]
[filtering-chart-data](https://filamentphp.com/docs/3.x/widgets/charts#filtering-chart-data) ?

**Reply by YANGJIAXUE2022510** [reply]
"I have already implemented this, thank you. I'm actually curious about the 'Stat' can only be one at a time. This way, I need many blocks to achieve the above."
![image](https://github.com/filamentphp/filament/assets/105269516/5a455296-8414-4088-94cb-f8bd5471bf40)
 "I'm currently writing it this way."
```
return [
            
            Stat::make('总消费', '277.83 USD')
            ->descriptionIcon('heroicon-m-arrow-trending-up')
            ->chart([7, 2, 10, 3, 15, 4, 17])
            ->color('success'),
            // Stat::make('总消费', Patient::query()->where('type', 'cat')->count())  , 
            // Stat::make('广告数量', Patient::query()->where('type', 'dog')->count()),
            // Stat::make('账户状态', Patient::query()->where('type', 'rabbit')->count()),
        ];
```
"Although the image and the code are completely unrelated, the image is what I've implemented, and the code is what I'm puzzled about."
